src/transformStateWithClones.js

```diff
@@ -65,7 +65,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const transformedStates = [];
+  const newState = { ...state };
+
+  for (let i = 0; i < transforms.length; i++) {
+    switch (transforms[i].operation) {
```

Comment:
you can desctucture
const {operation, properties} = transforms[I];
and then use it in code